packages/core/src/env.ts

```diff
@@ -1,208 +1,81 @@
-  }
-
-  if (getAIConfigInBoolean(MIDSCENE_USE_GEMINI)) {
-    return 'gemini';
-  }
-
-  if (getAIConfigInBoolean(MIDSCENE_USE_VL_MODEL)) {
-    return 'vl-model';
-  }
-
-  if (getAIConfigInBoolean(MIDSCENE_USE_VLM_UI_TARS)) {
-    return 'vlm-ui-tars';
-  }
-
-  return false;
-};
-
-export const getAIConfig = (
-  configKey: keyof ReturnType<typeof allConfigFromEnv>,
-): string | undefined => {
-  if (configKey === MATCH_BY_POSITION) {
-    throw new Error(
-      'MATCH_BY_POSITION is deprecated, use MIDSCENE_USE_VL_MODEL instead',
-    );
-  }
-
-  return getGlobalConfig()[configKey]?.trim();
-};
-
-export const getAIConfigInBoolean = (
-  configKey: keyof ReturnType<typeof allConfigFromEnv>,
-) => {
-  const config = getAIConfig(configKey) || '';
-  return /^(true|1)$/i.test(config);
-};
-
-export const getAIConfigInJson = (
-  configKey: keyof ReturnType<typeof allConfigFromEnv>,
-) => {
-  const config = getAIConfig(configKey);
-  try {
-    return config ? JSON.parse(config) : undefined;
-  } catch (error: any) {
-    throw new Error(
-      `Failed to parse json config: ${configKey}. ${error.message}`,
-      {
-        cause: error,
-      },
-    );
-  }
-};
-
-export const overrideAIConfig = (
-  newConfig: Partial<ReturnType<typeof allConfigFromEnv>>,
-  extendMode = false, // true: merge with global config, false: override global config
-) => {
-  for (const key in newConfig) {
-    if (typeof key !== 'string') {
-      throw new Error(`Failed to override AI config, invalid key: ${key}`);
-    }
-    if (typeof newConfig[key as keyof typeof newConfig] === 'object') {
-      throw new Error(
-        `Failed to override AI config, invalid value for key: ${key}, value: ${newConfig[key as keyof typeof newConfig]}`,
-      );
-    }
-  }
-
-  const currentConfig = getGlobalConfig();
-  globalConfig = extendMode
-    ? { ...currentConfig, ...newConfig }
-    : { ...newConfig };
+import {
+  ANTHROPIC_API_KEY,
+  AZURE_OPENAI_API_VERSION,
+  AZURE_OPENAI_DEPLOYMENT,
+  AZURE_OPENAI_ENDPOINT,
+  AZURE_OPENAI_KEY,
+  MATCH_BY_POSITION,
+  MIDSCENE_API_TYPE,
+  MIDSCENE_AZURE_OPENAI_INIT_CONFIG_JSON,
+  MIDSCENE_AZURE_OPENAI_SCOPE,
+  MIDSCENE_CACHE,
+  MIDSCENE_DANGEROUSLY_PRINT_ALL_CONFIG,
+  MIDSCENE_DEBUG_AI_PROFILE,
+  MIDSCENE_DEBUG_AI_RESPONSE,
+  MIDSCENE_DEBUG_MODE,
+  MIDSCENE_FORCE_DEEP_THINK,
+  MIDSCENE_LANGSMITH_DEBUG,
+  MIDSCENE_MCP_USE_PUPPETEER_MODE,
+  MIDSCENE_MODEL_NAME,
+  MIDSCENE_OPENAI_INIT_CONFIG_JSON,
+  MIDSCENE_OPENAI_SOCKS_PROXY,
+  MIDSCENE_REPORT_TAG_NAME,
+  MIDSCENE_USE_ANTHROPIC_SDK,
+  MIDSCENE_USE_AZURE_OPENAI,
+  MIDSCENE_USE_DOUBAO_VISION,
+  MIDSCENE_USE_GEMINI,
+  MIDSCENE_USE_QWEN_VL,
+  MIDSCENE_USE_VLM_UI_TARS,
+  MIDSCENE_USE_VL_MODEL,
+  OPENAI_API_KEY,
+  OPENAI_BASE_URL,
+  OPENAI_MAX_TOKENS,
+  OPENAI_USE_AZURE,
+  allConfigFromEnv,
+  getAIConfig,
+  getAIConfigInBoolean,
+  getAIConfigInJson,
+  overrideAIConfig,
+  vlLocateMode,
+} from '@midscene/shared/env';
+
+export {
```

Comment:
We can directly re-export

Reply:
Perhaps it's also possible to search for exactly how many places have used core/env, as there is also a re-export for this. Could it be considered to directly modify it